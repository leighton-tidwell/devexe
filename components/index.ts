export { Button } from './Button/Button';
export { Header } from './Header/Header';
export { Icon } from './Icon/Icon';
export { Logo } from './Logo/Logo';
export { Spinner } from './Spinner/Spinner';
export { Wrapper } from './Wrapper/Wrapper';
