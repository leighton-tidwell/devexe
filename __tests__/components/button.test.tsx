import { Button } from '@/components/index';
import { render, screen } from '../test-utils';

describe('Button', () => {
  it('should render a button with primary styles applied', () => {
    render(<Button>Hello World</Button>);

    expect(screen.getByRole('button')).toBeInTheDocument();
  });
});
