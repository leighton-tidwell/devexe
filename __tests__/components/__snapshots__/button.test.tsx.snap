// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button when the variant is cta should have correct styles 1`] = `
<div>
  <button
    aria-busy="false"
    class="css-oupfa2-Box"
    name="test-button"
    role="button"
  >
    <span
      class="css-1peva9n-Box"
    >
      Test
    </span>
  </button>
</div>
`;

exports[`Button when the variant is primary should have correct styles 1`] = `
<div>
  <button
    aria-busy="false"
    class="css-oei820-Box"
    name="test-button"
    role="button"
  >
    <span
      class="css-1peva9n-Box"
    >
      Test
    </span>
  </button>
</div>
`;
